Automattic/wp-calypso
# Add component GlobalStylesVariations in @automattic/global-styles

## Proposed Changes

This PR refactors the package `@automattic/design-preview` by moving its style variation preview to the package `@automattic/global-styles`. This way, `@automattic/global-styles` becomes the de facto package for rendering style preview for the following pages:

Design preview in the onboarding design picker:
![Screenshot 2023-03-27 at 9 54 04 PM](https://user-images.githubusercontent.com/797888/227959942-c4211f73-392b-415e-bd59-4f2a7324503c.png)

Pattern Assembler.
![Screenshot 2023-03-27 at 9 54 36 PM](https://user-images.githubusercontent.com/797888/227960372-b9a57a6d-f3c9-4381-a0d2-4e1dc11f4f74.png)
![Screenshot 2023-03-27 at 9 54 44 PM](https://user-images.githubusercontent.com/797888/227960388-3dadf500-55f6-4a88-ba3b-82a6dcc294fb.png)

Theme Detail page.
![Screenshot 2023-03-27 at 9 55 30 PM](https://user-images.githubusercontent.com/797888/227960328-5d441424-6df7-4978-b9cc-988a1451154c.png)

## Testing Instructions



* Head to the pages mentioned above:
  * Onboarding design picker.
  * Pattern Assembler.
  * Theme Detail page.
* Ensure that the style previews work as before.

## Pre-merge Checklist
